examples/Cylinder/input_test.i3d

```diff
@@ -37,19 +37,19 @@ iin = 1               ! Inflow conditions (1: classic, 2: turbinit)
 re  = 300. ! nu=1/re (Kinematic Viscosity)
 u1  = 1.              ! u1 (max velocity) (for inflow condition)
 u2  = 1.              ! u2 (min velocity) (for inflow condition)
-init_noise  = 0.125   ! Turbulence intensity (1=100%) !! Initial condition
+init_noise  = 0.001   ! Turbulence intensity (1=100%) !! Initial condition
 inflow_noise = 0.0    ! Turbulence intensity (1=100%) !! Inflow condition
 
 ! Time stepping
 dt = 0.0075              ! Time step
 ifirst = 1            ! First iteration
-ilast =  100     ! Last iteration
+ilast =  4001     ! Last iteration
```

Comment:
Why are we changing the test input file with a much longer duration instead of creating a new input file ?

Reply:
Will change it back to 100. I had tested the code for a longer time.